Makefile

```diff
@@ -139,7 +142,22 @@ TMPDIR ?= $(if $(shell if [ -d /scratch ] ; then echo true; fi),/scratch/$(USER)
 
 ### Default tag
 help:
-	@echo "Targets:"
+	@echo
+	@echo "This is the primary Makefile for both an archive file based setup"
+	@echo "(i.e. created from daikon-<version>.zip) and for a repository"
+	@echo "based setup (i.e. git cloned from //github.com/codespecs/daikon)."
+	@echo "This is primarily for users who wish to customize or extend DAIKON."
+	@echo
+	@echo "These first two targets are only for the archive version:"
+	@echo " install                  -- build additional tools and kvasir"
```

Comment:
In the Linux world, "make install" means to place the program's binary somewhere on your path.  This seems different than that, which I think will be confusing to users.